### ISSUE TITLE
Before insert hook?

### ISSUE THREAD

**Issue by morgondag** [ef4/liquid-fire#278]
Tried to find some example with the new layout for liquid custom transitions.
Is there no longer a hook before the new view is inserted?

##### version 0.17.0 
 We hade an exposed area  **before the newview is inserted**
```js
import { /*isAnimating, finish, timeSpent,*/ animate, stop } from "liquid-fire";
export default function(oldView, insertNewView/*, opts*/) {
	// do something here before the new view is inserted
	// the url is the old url
	return insertNewView().then(function(newView) {
		// animate here
		// the url is the new url
	}
}
```
##### version 0.19.3
The newView is already inserted and the url is already updated before the function is exposed
```js
import {animate, stop, Promise} from "liquid-fire";
export default function() {
	// the view is already inserted
	// the url is the new url
}
```

for example i'm using a transition like this:

```js
    this.transition(
        this.fromRoute('app/settings/about'),
        this.toRoute('app/settings'),
        this.use('toRightCallback')
    );
```

Or is there a new elegant way to do something when the transition started but before the new view is inserted?

**Comment by ef4** [comment]
Yes, the API has been simplified. The new view is already inserted, but hidden. You can choose when to reveal it.

But liquid-fire has never blocked the URL from changing.

**Comment by morgondag** [comment]
agree it have never blocked the url that is good. 
I want to run some code the newview is inserted but the transition has started.

so there is no hook available to use before the newview is inserted at all?
